src/karmen_frontend/src/actions/printers/reading.js

```diff
@@ -255,6 +261,9 @@ export const getWebcamSnapshot = createHttpAction(
         };
       });
     }).catch((err) => {
+      if (err.message === "organization mismatch") {
+        return;
+      }
```

Comment:
It's better to match errors on other attributes than messages, right? (object type, ...)

Reply:
Changed to custom error class

Reply:
Used custom error class.